src/EnergyPlus/Construction.hh

```diff
@@ -100,7 +100,7 @@ namespace Construction {
         std::array<Real64, MaxCTFTerms> CTFOutside{};   // Outside or X terms of the CTF equation
         std::array<Real64, MaxCTFTerms> CTFSourceIn{};  // Heat source/sink inside terms of CTF equation
         std::array<Real64, MaxCTFTerms> CTFSourceOut{}; // Heat source/sink outside terms of CTF equation
-        Real64 CTFTimeStep;                             // Time increment for stable simulation of construct (could be greater than TimeStep)
+        Real64 CTFTimeStep = 0.0;                       // Time increment for stable simulation of construct (could be greater than TimeStep)
```

Comment:
Seems no problem.  I can imagine this sneaking by, primarily in unit tests, and causing funny business.